packages/button/src/Button.tsx

```diff
@@ -348,8 +375,11 @@ const Button = React.forwardRef((props: ButtonProps, forwardRef) => {
           //
           // https://www.w3.org/TR/css-flexbox-1/#painting
           z-index: 1;
+          display: flex;
+          align-items: center;
```

Comment:
Given we have several CSS properties here now, we should move this from being an inline style declaration to a variable.

Additionally, would `display: inline-flex` be a better change from the default of `display: inline`? Or is there a particular reason we need this to have block sizing?